Sources/AsyncHTTPClient/HTTPClient.swift

```diff
@@ -276,19 +276,23 @@ public class HTTPClient {
         public var timeout: Timeout
         /// Upstream proxy, defaults to no proxy.
         public var proxy: Proxy?
+        /// Enables ignore NIOSSLError.uncleanShutdown when state is .head or .body, defaults to `false`.
```

Comment:
I think we need a better description. Maybe: "Ignore TLS unclean shutdown errors".

Reply:
It was updated.